docs/server/tutorials/tlsssl.md

```diff
@@ -10,7 +10,23 @@ If you intend to make your Sourcegraph instance accessible on the Internet or an
 
 Sourcegraph will use [Let's Encrypt](https://letsencrypt.org/) by default if the following conditions are met:
 
-- Your `appURL` site configuration option begins with `https://...`, and the host is reachable on port 80 and port 443.
+- Your `appURL` site configuration option begins with `https://...`.
+- The host is reachable on port `80` and port `443`.
+
+  - 🚨 [Lets Encrypt requires that port `80` be reachable in order prove that you own your domain](https://letsencrypt.readthedocs.io/en/latest/challenges.html#http-01-challenge). If port `80` is unreachable, HTTPS will fail with errors such as the following:
```

Comment:
I think it would be better to put this warning + the errors at the end of the section here. Otherwise, it is hard to read the bullet points here about `if the following conditions are met:` without losing your train of thought and/or not realizing more bullets come after.